lidar_point_pillars/package.xml

```diff
@@ -13,4 +13,6 @@
   <depend>pcl_ros</depend>
   <depend>roscpp</depend>
   <depend>roslib</depend>
+  <depend>tvm_utility</depend>
+  <depend>tvm_vendor</depend>
```

Comment:
`tvm_utility` implicitly relies on `tvm_vendor` so you can remove `tvm_vendor` here. You don't have to specify every dependency down the chain, just the direct dependencies.

Reply:
Done